recipes/vtk/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "7.0.0" %}
+{% set minor_version = ".".join(version.split(".")[:2]) %}
+
+package:
+  name: vtk
+  version: {{ version }}
+
+source:
+  url: http://www.vtk.org/files/release/{{ minor_version }}/VTK-{{ version }}.tar.gz
+  fn:  VTK-{{ version }}.tar.gz
+  sha256: 78a990a15ead79cdc752e86b83cfab7dbf5b7ef51ba409db02570dbdd9ec32c3
+
+build:
+  number: 0
+  features:
+    - vc9                                  # [win and py27]
+    - vc10                                 # [win and py34]
+    - vc14                                 # [win and py35]
+
+requirements:
+  build:
+    - freeglut                             # [linux and (py27 or py34 or py35)]
```

Comment:
These are the only Python's we build for so the selector can just be `# [linux]`.